app/Livewire/Project/Service/StackForm.php

```
@@ -39,19 +50,44 @@ protected function messages(): array
             ]
         );
     }
 
     public $validationAttributes = [];
 
+    /**
+     * Sync data between component properties and model
+     *
+     * @param  bool  $toModel  If true, sync FROM properties TO model. If false, sync FROM model TO properties.
+     */
+    private function syncData(bool $toModel = false): void
```

PHPMD (2.15.0)

The method syncData has a boolean flag argument $toModel, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)